mlir/lib/Tools/mlir-opt/MlirOptMain.cpp

```diff
@@ -241,23 +296,23 @@ class DiagnosticFilter : public ScopedDiagnosticHandler {
     setHandler([verbosityLevel, showNotes](Diagnostic &diag) {
       auto severity = diag.getSeverity();
       switch (severity) {
-      case DiagnosticSeverity::Error:
+      case mlir::DiagnosticSeverity::Error:
```

Comment:
Why were these changes needed?

Reply:
Conflict with the LLVM class of the same name I believe (there is an added include to LLVM)